### PR TITLE
Add creator address in contract

### PR DESCRIPTION
**Pull request by 0xaptosj**
**Usage**

Find your Petra address you want to use on the UI, set `initial_creator_address` in `publish.js` with it.
This approach avoids user having to call a separate function to set creator after publishing module because we set it in `init_module()`.

**Changes**
- Remove compile script because we don't need it, both test and publish will compile automatically.
- Set `initial_creatpr_address` in test and publish scripts.

### CODE REVIEW

**Review by 0xmaayan — COMMENTED** [review]
Let's keep the compile script

File: templates/digital-asset-template/scripts/move/publish.js [hunk 1]
```diff
@@ -14,8 +14,10 @@ async function publish() {
     packageDirectoryPath: "move",
     namedAddresses: {
       launchpad_addr: accountAddress,
+      // This is the address you want to use to create collection with, e.g. an address in Petra so you can create collection in UI using Petra
+      initial_creator_address: "to_fill",
       // Please find it on the network you are using
-      // This is my own deployed version on testnet
+      // This is testnet deployment
```
Comment (0xmaayan) [inline templates/digital-asset-template/scripts/move/publish.js:20]: can we explain a bit more what is this address?

Comment (0xaptosj) [reply templates/digital-asset-template/scripts/move/publish.js]: done

**Comment by 0xaptosj** [comment]
> Let's keep the compile script

restored

**Comment by 0xmaayan** [comment]
There some code we can clean up following this change, for example https://github.com/aptos-labs/create-aptos-dapp/blob/main/templates/digital-asset-template/frontend/pages/CreateCollection.tsx#L198-L212 

Also, why not set the `creator_address` on `.env` and get it from there when we want to publish the module?

**Pull request merged**